sched/worker/worker.go

```diff
@@ -5,44 +5,19 @@ import (
 	"github.com/scootdev/scoot/sched"
 )
 
-// Create a WorkerController that talks to node
-type WorkerFactory func(node cluster.Node) WorkerController
+// Create a Worker controller that talks to node
+type WorkerFactory func(node cluster.Node) Worker
 
-// WorkerController lets the Scheduler control a worker.
-type WorkerController interface {
+// Worker controller gives the Scheduler a generic way to complete work.
+type Worker interface {
 	// TODO(dbentley): include more info on positive results
 	RunAndWait(task sched.TaskDefinition) error
 }
 
 type workerNode struct {
```

Comment:
I'll remove it for now. In order for work to flow through though, we need a way to convert Node.Id to an address etc. We may have decided that can all go in cluster.Node, I don't remember.